pygeoapi/process/shapely_functions.py

```diff
@@ -0,0 +1,139 @@
+# =================================================================
+#
+# Authors: Krishna Lodha <krishna@rottengrapes.com>
+#
+# Copyright (c) 2023 Krishna Lodha
+#
+# Permission is hereby granted, free of charge, to any person
+# obtaining a copy of this software and associated documentation
+# files (the "Software"), to deal in the Software without
+# restriction, including without limitation the rights to use,
+# copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following
+# conditions:
+#
+# The above copyright notice and this permission notice shall be
+# included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+# OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+# HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+# WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+# =================================================================
+
+import logging
+
+from pygeoapi.process.base import BaseProcessor, ProcessorExecuteError
+import shapely 
+
+LOGGER = logging.getLogger(__name__)
+
+#: Process metadata and description
+PROCESS_METADATA = {
+    'version': '0.2.0',
+    'id': 'geom-contains',
+    'title': {
+        'en': 'Shapely Contains functon',
+        'fr': 'Shapely Contains functon'
+    },
+    'description': {
+        'en': 'An example process that takes 2 geometries (A & B) in WKT Format '
+              'and returns Boolean representing if B is completely withing A',
+        'fr': 'Un exemple de processus qui prend 2 géométries (A et B) au format WKT '
+              '''et renvoie une valeur booléenne représentant si B est complètement à l'intérieur de A'''
+    },
+    'jobControlOptions': ['sync-execute', 'async-execute'],
+    'keywords': ['contains', 'shapely', ],
+    'links': [{
+        'type': 'text/html',
+        'rel': 'about',
+        'title': 'information',
+        'href': 'https://example.org/process',
```

Comment:
Point to https://shapely.readthedocs.io/en/stable/manual.html#object.contains in the `href`.